source/scenes/external/SelectAccounts/index.tsx

```diff
@@ -125,10 +125,12 @@ const SelectAccounts = () => {
       controller.dapp.fromUserConnectDApp(origin, current, network, selectedAccounts);
       const background = await browser.runtime.getBackgroundPage();
 
+      const {windowId} = queryString.parse(window.location.search);
+
       background.dispatchEvent(
-        new CustomEvent('connectWallet', { detail: { hash: window.location.hash, accounts: selectedAccounts } })
+        new CustomEvent('connectWallet', { detail: { windowId, accounts: selectedAccounts } })
```

Comment:
`window.location.hash` was always empty here so I added `windowId` as a query param instead. It seems odd to use a separate system for the hash/windowId anyways - might as well just make them all queryParams.